docs/en/guides/pipeline-integrations/development/sgtk-developer-app.md

```diff
@@ -0,0 +1,257 @@
+By using this app you get all the standard Toolkit boilerplate code set up for you, and a basic example GUI.
+
+![Toolkit starter app default GUI](./images/starter-app.png)
+
+To use it, you can either fork the git repository and then clone it to your local dev area on disk, 
+or if you don't want to use git source control at this stage, you can just download the files from GitHub as a zip file, and unzip them locally (you can always set up a git repository later).
+Either way, the goal is to have a local copy of the starter app code so you can start making changes.
+
+## Part 3: Adding the app to your config
+We recommend reading the "[Adding an app](../getting-started/installing_app.md)" guide, as it explains in more detail how to add an app to your configuration.
+
+When adding the app to your config, you need to consider where your app will be used, i.e. perhaps only in Nuke or in multiple different software, or standalone from Shotgun Desktop.
+You also need to think about the context that your app will depend on.
+For example, can your app only run when you know the task the user is working on, or can it run with just the project known?
+Knowing this will dictate which environment YAMLs and engines you need to add your app settings to.
+
+If you're not sure right now, it's a good idea to start by adding it to the `tk-shell` engine in the project environment, that way you can [run it from your IDE](./sgtk-developer-bootstrapping.md) or via the command line with the tank command if you have a [centralized config](https://developer.shotgunsoftware.com/tk-core/initializing.html#centralized-configurations), which will make it quicker to develop with.
+
+To start with you will use a [dev descriptor](https://developer.shotgunsoftware.com/tk-core/descriptor.html#pointing-to-a-path-on-disk) for the location of your app.
+
+```yaml
+tk-multi-starterapp:
+  location:
+    type: dev
+    path: /path/to/source_code/tk-multi-starterapp
+```
+This instructs Toolkit to load the app code directly from disk in the given location, which is great for development, where you want to change the code all the time.
+Later when you add the app to your production config, you may want to use a different descriptor.
+
+Now you've added the app to an environment, you should be able to go ahead and launch it. How you launch it will depend on the engine and environment you defined it in.
+
+## Part 4: Developing the app
+
+You are now all set to start developing your app!
+
+### Anatomy of the Template Starter App
+
+The [template starter app](https://github.com/shotgunsoftware/tk-multi-starterapp) contains the following basic structure:
+
+![](./images/tk-multi-starterapp-directory.png)
+
+- **app.py** - The app entry point and menu registration can be found in the `app.py` file. This is where you typically set up your classes and get things initialized and get menu items registered.
```

Comment:
I'd make that last sentence a list: "... set up your classes, get things initialized, and get menu items registered."